docs/using-kontena/volumes.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Volume Management
+---
+
+# Volumes
+
+Kontena provides capability to manage persistent data using volumes.
```

Comment:
Kontena 1.2 provides ***experimental*** support for managing persistent service data using Docker volumes. The exact details of how these Kontena volumes are managed may still change as the implementation evolves.